client/next.js

```diff
@@ -10,18 +10,22 @@ const {
   __NEXT_DATA__: { component, errorComponent, props, ids, err }
 } = window
 
-const Component = evalScript(component).default
-const ErrorComponent = evalScript(errorComponent).default
+document.addEventListener('DOMContentLoaded', () => {
+  const Component = evalScript(component).default
+  const ErrorComponent = evalScript(errorComponent).default
 
-export const router = new Router(window.location.href, {
```

Comment:
Additionally, I hope this feature(this export) is not used in our core code and is documented in the README. (or in examples).
Then removing this has no effect at all.